flux_sdk/fertility_carrot/capabilities/update_eligibility_data/interface.py

```diff
@@ -0,0 +1,29 @@
+from abc import ABC, abstractmethod
+
+from flux_sdk.flex.capabilities.update_contributions.data_models import EmployeeEligibilityRecord
+from flux_sdk.flux_core.data_models import CompanyInfo, File
+
+
+class UpdateEligibilityData(ABC):
```

Comment:
```suggestion
class ReportEligibilityData(ABC):
```
IIUC, we're reporting eligibility data to the 3rd party.  We want to reserve the update verb for when we're making changes in Rippling in response to data/actions from the 3rd party.

Reply:
Updated. Thanks.